rebar.config

```diff
@@ -1,4 +1,4 @@
-{erl_opts, [debug_info]}.
+{erl_opts, [debug_info, {platform_define, "^[0-9]+", namespaced_types}]}.
```

Comment:
why are you calling this "namespaced_types"? You're not using this as logic to control the use of namespaced types, so it's misleading.

Moreover, why wouldn't we just always try `strong_rand_bytes` here? it's not like that function didn't exist in older versions of Erlang.